libraries/js/mutable/array.effekt

```diff
@@ -8,6 +8,29 @@ extern type Array[T]
 
 def emptyArray[T](): Array[T] = emptyArray(0)
 
+//initialize array with default values
+def initArray[T](size: Int, default: T) : Array[T] = {
```

Comment:
Could we call it `filledArray` or `arrayFromValue` in accordance with the other two?

Reply:
done